src/wechaty.ts

```diff
@@ -202,6 +215,38 @@ export class Wechaty extends EventEmitter implements Sayable {
     return this.globalInstance
   }
 
+  /**
+   * @param   {WechatyPlugin | WechatyPlugin[]} plugins      - The plugins you want to use
+   *
+   * @return  {Wechaty}                                      - this for chaining,
+   *
+   * @desc
+   * For wechaty ecosystem, allow user to define a 3rd party plugin for the all wechaty instances
+   *
+   * @example
+   * // Report all chat message to my server.
+   *
+   * Class WechatyReportPlugin {
+   *   constructor(options) {
+   *     this.options = options;
+   *   }
+   *   install(bot) {
+   *     bot.on('message', message => http.post(this.options.url, { data: message }))
+   *   }
+   * }
+   *
+   * bot.use(new WechatyReportPlugin({ url: 'http://somewhere.to.report.your.data.com' });
+   */
+  public static use (
```

Comment:
It's great to be able to set multiple plugins at once.

Instead of `use([])`, how about let's be able to call `use(...plugins)` , so that it will be more concise?

Reply:
Awesome!